index.d.ts

```diff
@@ -0,0 +1,18 @@
+import { cleanup, act, RenderOptions, RenderResult } from 'react-testing-library'
+
+export function renderHook<P extends any, T extends (...args: [P]) => any>(
+  callback: (_: P) => ReturnType<T>,
```

Comment:
Where does `ReturnType` come from?

Reply:
Nevermind, I found [something about it in the TS docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html)